v63002/u_inref/gtm8616.csh

```diff
@@ -0,0 +1,26 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+set t = `date +"%b %e %H:%M:%S"`
+sleep 1
+# Using .outx because expecting a warning about an argumentless MUPIP RUNDOWN
+echo "# Running argumentless MUPIP RUNDOWN and checking syslog"
+echo ""
+$MUPIP RUNDOWN >& rundown.outx
+$gtm_tst/com/getoper.csh "$t" "" getoper.txt
+echo "# Syslog message:"
+$grep "process id" getoper.txt |& sed 's/.*%YDB-I-MURNDWNARGLESS/%YDB-I-MURNDWNARGLESS/'
```

Comment:
I would like the pid and username verified too. Not just the path. For pid, start the process in the background (which means you need to wait for this to die at the end) and $! will give you the pid.